apps/coscrad-frontend/src/components/term-of-the-day/term-of-the-day.presenter.tsx

```diff
@@ -0,0 +1,61 @@
+import { ICategorizableDetailQueryResult, ITermViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Card, Divider } from '@mui/material';
+
+type HasTermOfTheDay = {
```

Comment:
```ts
type MonthAndDay = {
   month: 'January' | 'February' | .... 'December'; // might want enum here, will need to map to numeric indices
   day: number; // should be an integer
}

// These types might be defined in the config and just reused here.

interface TermOfTheDayPresenterProps{
    termOfTheDayConfig: TermOfTheDayConfig,
    date: MonthAndDate;
}
```

Reply:
Alternatively, you might consider doing the search at the higher level and just passing in:
```ts
interface TermOfTheDayPresenterProps{
    term: ICategorizableDetailQueryResult<ITermViewModel>;
    date: MonthAndDate;
}
```

In fact, I prefer this, because there will be the 'loadable' behaviour which should be handled at the level of the container.